Dockerfile

```diff
@@ -0,0 +1,21 @@
+from ubuntu:bionic
+
+RUN apt-get update && apt-get install locales=2.27-3ubuntu1 git python3.6 python3-pip -qy && \
```

Comment:
DO you remember why locales is pinned? I found this PR where I unpinned the rest, but not locales, maybe because unpinning it broke something?

Reply:
We can probably unpin it.  I tend to pin things so the container doesn't stop building at some point, but I think that's probably a habbit caused by not having a daily build.  I'll remove it.